notebooks/img/leafmap_mess.ipynb

```diff
@@ -1828,13 +219,22 @@
     "\n",
     "hover_style = {\"fillOpacity\": 0.7}\n",
     "\n",
-    "gdf4001_geojson = leafmap.gdf_to_geojson(gdf_4001, epsg=\"4326\")\n",
-    "gdf1001_geojson = leafmap.gdf_to_geojson(gdf_1001, epsg=\"4326\")\n",
+    "# useless because it shows the data already used in the background map\n",
+    "#style1 = {\n",
+    "#    \"stroke\": True,\n",
+    "#    \"color\": \"#0000ff\",\n",
+    "#    \"weight\": 2,\n",
+    "#    \"opacity\": 1,\n",
+    "#    \"fill\": True,\n",
+    "#    \"fillColor\": \"#0000ff\",\n",
+    "#    \"fillOpacity\": 0.1,\n",
+    "#}\n",
+    "#gdf4001_geojson = leafmap.gdf_to_geojson(gdf_4001, epsg=\"4326\")\n",
+    "#merc_map.add_geojson(gdf4001_geojson, layer_name=\"4001 Collection\", style = style1, hover_style = hover_style)\n",
```

Comment:
Should this get removed before merge?

Reply:
Yes, I am always a bit sentimental and afraid of throwing away code which works but you are right, that pollutes the readibility of the notebook and I removed all the useless references to the collection 4001.